### PR TITLE
[FIX] l10n_do: Updating tax to match with new regulations

### PR DESCRIPTION
**Pull request by akevinieron**
Before this PR there was an outdated tax. This PR add the new tax amount according to the new legislation.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by bfr-o — APPROVED** [review]
LGTM. Can you link to a document where we see that the legislation changed, just for traceability  purpose ?

**Comment by akevinieron** [comment]
Here links to both official documents:
N08-10 | Old (The change is in last paragraph, first page) [[https://dgii.gov.do/legislacion/normasGenerales/Documents/NG%20sobre%20Impuesto%20sobre%20Transferencias%20de%20Bienes%20Industrializados%20y%20Servicios%20(ITBIS)/norma08-10.pdf](url)]

N05-19 | New (Change in first line, page 7)
[[https://dgii.gov.do/legislacion/normasGenerales/Documents/NG%20sobre%20Comprobantes%20Fiscales/Norma05-19.pdf](url)]

File: addons/l10n_do/data/account.tax.template.xml [hunk 1]
```diff
@@ -923,12 +923,12 @@
             ]"/>
         </record>
 
-        <record id="ret_75_tax_nonformal" model="account.tax.template">
+        <record id="ret_100_tax_nonformal" model="account.tax.template">
```
Comment (bfr-o) [inline addons/l10n_do/data/account.tax.template.xml:926]: Can't change xml_id of a record in stable version. You should keep the xml_id in this version and rename it only in master (in the automatic fw-port or in a different PR).

**Review by bfr-o — CHANGES_REQUESTED** [review]
Sorry, I didn't see that it was for a stable version. See https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean for more details about what is allowed or not in stable versions.

File: addons/l10n_do/data/account.tax.template.xml (hunk 1 above)
Comment (akevinieron) [reply addons/l10n_do/data/account.tax.template.xml]: @bfr-o Done!! Thanks for the info.

**Review by bfr-o — APPROVED** [review]

**Comment by bfr-o** [comment]
@akevinieron After talking with my colleagues, when a tax changes, it's better to create a new tax and make the old tax inactive. That way, it's still available, in case one would want to change something in a previous period.

**Comment by akevinieron** [comment]
@bfr-o All clear!!

**Comment by william-andre** [comment]
@oco-odoo Shouldn't this be done in saas-12.3?

**Comment by jco-odoo** [comment]
@gustavovalverde fyi

**Comment by oco-odoo** [comment]
@akevinieron We could indeed directly do that in saas-12.3, in order to have as many people as possible benifiting from the patch. Also, don't forget to squash your commits, and add the missing [CLA] tag for the adding of the .md file (see https://www.odoo.com/documentation/14.0/developer/reference/guidelines.html?highlight=guidelines#commit-message-structure). Keep 1 commit for the CLA, and 1 commit for the rest.

**Comment by gustavovalverde** [comment]
@jco-odoo I've just reviewed it, this changes can be merged

**Review by gustavovalverde — APPROVED** [review]

**Comment by oco-odoo** [comment]
@akevinieron Squash your commits (the way I explained earlier), then we can merge :)

**Comment by C3POdoo** [comment]
Dear @akevinieron,

Thank you for your contribution but the version 13.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**